azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerPropertyResolutionTest.java

```diff
@@ -107,15 +110,15 @@ private void assertProperties(final boolean isAzkabanFlowVersion20) throws Excep
     Assert.assertEquals("shared1", job2Props.get("props1"));
     Assert.assertEquals("job2", job2Props.get("props2"));
     Assert.assertEquals("moo3", job2Props.get("props3"));
-    Assert.assertEquals("job7", job2Props.get("props7"));
+    Assert.assertEquals("flow7", job2Props.get("props7"));
     Assert.assertEquals("flow5", job2Props.get("props5"));
     Assert.assertEquals("flow6", job2Props.get("props6"));
     Assert.assertEquals("shared4", job2Props.get("props4"));
     Assert.assertEquals("shared8", job2Props.get("props8"));
 
     // Job 1 is inside another flow, and is nested in a different directory
     // The priority order should be:
-    // job1->innerflow->job2.output->flow.overrides->job1 shared props
+    // job1->innerflow->job2.output->job1.sharedprops->flow.overrides
```

Comment:
@juhoautio with `EXECUTOR_PROPS_RESOLVE_OVERRIDE_EXISTING_ENABLED` set to true I believe this comment  (and similar below) is not correct, flow overrides now will win over everything so it should be: 
`flow.overrides->job1->innerflow->job2.output->job1.sharedprops`